src/api/cow/index.ts

```diff
@@ -166,6 +166,24 @@ export class CowApi<T extends ChainId> {
     }
   }
 
+  async getTxOrders(txHash: string): Promise<OrderMetaData[]> {
+    log.debug(`[api:${this.API_NAME}] Get tx orders for `, this.chainId, txHash)
+
+    try {
+      const response = await this.get(`/transactions/${txHash}/orders`)
+
+      if (!response.ok) {
+        const errorResponse: ApiErrorObject = await response.json()
+        throw new OperatorError(errorResponse)
```

Comment:
I know there are other methods that do the same, but isn't this error going to be swallowed by the catch?

Reply:
@ramirotw Indeed. I made a change to properly handle the different types of errors. If you are ok with this, we could just add a new ticket to update the rest of the methods.

Reply:
This type of error handling (response does not contain json code) has been updated recently on cowswap.

I suggest taking a look there to update it here as well. Not part of this PR, but as a follow up task with lower priority.

Reply:
@alfetopito you mean this? https://github.com/gnosis/cowswap/blob/baf229374a68c207f1efbb1fb0279bc8a630405a/src/custom/api/gnosisProtocol/api.ts#L278

Reply:
More specifically this bit where we check response's content type https://github.com/gnosis/cowswap/blob/baf229374a68c207f1efbb1fb0279bc8a630405a/src/custom/api/gnosisProtocol/api.ts#L283-L288

The whole function doesn't make sense as we don't want to include sentry tracking in a lib